### DISCUSSION TITLE
3rd MHQ Vehicle

### DISCUSSION THREAD

**Discussion by AssadRajab** [Xeno69/Domination#630] [Q&A] [answered]
Hi, is it possible to have (for example) a 3rd MHQ vehicle? I tried to place the variable name (d_vec_mhq_3) according to the script. But it was not recognized. 

Also, where are the actions of the whiteboard (where the teleport option is) in the base defined?
I tried to look it up in the files but I did not see it anywhere.

**Comment by Xeno69** [comment, marked as answer]
Don't use d_vec_mhq_3 as var name but simply

d_mhq_add_x

Where  x is a unique number or char.

That's all you have to do, it will add that vehicle as another MHQ automatically.

Check also
https://github.com/Xeno69/Domination/blob/master/co30_Domination.Altis/server/fn_gmcwgextra.sqf

**Reply by AssadRajab** [reply]
Great, that's an excellent way of thinking. Thank you.
